kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -42,17 +43,14 @@ def add_extra_output_topic(self, topic_name: str, role: str) -> None:
 
     @override
     def get_helm_chart(self) -> str:
-        return f"{self.config.streams_bootstrap_helm_config.repository_name}/{AppType.PRODUCER_APP.value}"
+        return f"{self.repo_config.repository_name}/{AppType.PRODUCER_APP.value}"
```

Comment:
`self.repo_config` can be `None`

Reply:
The field `repo_config` has a default value in the `KafkaApp` class. How can this be `None`?

Reply:
ah, I see. Please override the type annotation in `KafkaApp`
```python
repo_config: HelmRepoConfig = HelmRepoConfig(...
```

otherwise the type checker infers it from `KubernetesApp` as `HelmRepoConfig | None`